### DISCUSSION TITLE
Deploying with actions/deploy-pages timeouts

### DISCUSSION THREAD

**Discussion by sharkudev** [community/community#67067] [Actions]
### Select Topic Area

Question

### Body

Current status: deployment_in_progress
Current status: deployment_in_progress
Current status:
Timeout reached, aborting!
Error: Timeout reached, aborting!

**Comment by fardinahosancse** [comment]
github-pages:
    environment:
      name: github-pages
      url: ${{ steps.deployment.outputs.page_url }}
    runs-on: ubuntu-22.04
    needs: test
    steps:
      - name: Deploy to GitHub Pages
        id: deployment
        uses: actions/deploy-pages@v2
        timeout-minutes: 1

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
